Modules/@babylonjs/react-native-windows/windows/BabylonReactNative/BabylonReactNative.vcxproj

```diff
@@ -115,6 +124,31 @@
       <SubSystem>Console</SubSystem>
       <GenerateWindowsMetadata>true</GenerateWindowsMetadata>
       <ModuleDefinitionFile>BabylonReactNative.def</ModuleDefinitionFile>
+      <!-- Library directories when building in repo -->
+      <!-- Note: you can add BabylonNative component build folders here as needed -->
+      <AdditionalLibraryDirectories Condition="'$(BabylonNativeBuildDir)' != ''">$(BabylonNativeBuildDir)\Dependencies\arcana.cpp\$(Configuration);%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>

```

Comment:
>AdditionalLibraryDirectories [](start = 7, length = 28)

nit: MSBuild supports multi-line properties, you don't need to duplicate this over and over. Just do something like:

```
<AdditionalLibraryDirectories Condition="'$(BabylonNativeBuildDir)' != ''">
    $(BabylonNativeBuildDir)\Dependencies\bgfx.cmake\$(Configuration);
    $(BabylonNativeBuildDir)\Dependencies\glslang\glslang\$(Configuration);
    ...
    %(AdditionalLibraryDirectories)
</AdditionalLibraryDirectories>
```

Reply:
i feel like i had originally tried this but hit issues. I'll look into this again

Reply:
this should now be updated